CMakeLists.txt

```diff
@@ -2,8 +2,8 @@ cmake_minimum_required(VERSION 3.14)
 include(GNUInstallDirs)
 project(hiredis-cluster)
 
-option(DOWNLOAD_HIREDIS "Download the dependency hiredis from GitHub" ON)
-option(ENABLE_SSL       "Enable SSL/TLS support" ON)
+option(DOWNLOAD_HIREDIS "Download the dependency hiredis from GitHub" OFF)
+option(ENABLE_SSL       "Enable SSL/TLS support" OFF)
```

Comment:
The defaults was added by us and set to make it easy to build when developing. ENABLE_SSL and DISABLE_TESTS exists in hiredis aswell and there they are default off, so this makes the SSL option coherent.
The default value for download options can be discussed, having it ON makes us use a known hiredis version as base, and should be known to work since we run tests on that. At the same time having all options as FF is a good default..

Reply:
Put tje option changes in an own commit, but landed in having the download option as on by default, and using redis v1.0.0 as the baseline.

Reply:
Sounds good.